frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -39,6 +40,7 @@ local MODE = {
     wifi_status = 10,
     book_title = 11,
     book_chapter = 12,
+    bookmarks_count = 13,
```

Comment:
Please also change the variable name while there's no legacy yet. ^_^
```suggestion
    bookmark_count = 13,
```